ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -169,9 +169,14 @@ private void onChunkOperationComplete(GetChunk chunk) {
         // chunk retrievals and channel writes will need to happen and for that, this operation needs the GetManager to
         // poll it periodically. If any exception is encountered while processing subsequent chunks, those will be
         // notified during the channel read.
-        getBlobResult = getOperationException() == null ? new GetBlobResult() : null;
-        operationCompleteCallback
-            .completeOperation(operationFuture, operationCallback, getBlobResult, getOperationException());
+        routerMetrics.getBlobOperationLatencyMs.update(time.milliseconds() - submissionTimeMs);
```

Comment:
Just to confirm that the operation latency is between the submission time and the time the first chunk received.